src/Umbraco.Core/Services/IPropertyValidationService.cs

```diff
@@ -25,5 +25,9 @@ public interface IPropertyValidationService
             string validationRegExp,
             string isRequiredMessage,
             string validationRegExpMessage);
+
+        public IEnumerable<ValidationResult> ValidatePropertyValue(
```

Comment:
```suggestion
        IEnumerable<ValidationResult> ValidatePropertyValue(
```

Reply:
Perhaps add an XML header comment to this and the other method missing it in this interface too.